acceptance/tests/invalid_ssl_config.rb

```diff
@@ -59,8 +59,8 @@
 on agent1, puppet('resource service pxp-agent ensure=running')
 retry_on(agent1, "grep 'TLS handshake failed' #{logfile(agent1)}", {:max_retries => 30,
                                                              :retry_interval => 1})
-assert(on(agent1, "grep 'retrying in' #{logfile(agent1)}"),
-       "pxp-agent should log that it will retry connection")
+retry_on(agent1, "grep 'retrying in' #{logfile(agent1)}", {:max_retries => 10,
```

Comment:
@james-stocks I ran into a race condition on AIX where this assertion would fail because the check had happened a couple of seconds before the expected line was written to the log.